WebApp/src/Class/PlayerRace.tsx

```diff
@@ -0,0 +1,10 @@
+class PlayerRace implements IRace {
```

Comment:
look at the files in the Race folder. more info is available (alignment and such)